### ISSUE TITLE
will it work for third order categorical features interaction

### ISSUE THREAD

**Issue by Sandy4321** [EthanRosenthal/newtonfm#3]
Great code,  thanks !

Plese help to understand
1
will it work for third order categorical features interaction ?
2
will it run on Windows computer ?

3
will it work for sparse data ?

**Comment by EthanRosenthal** [comment]
No, I don't think this supports third order feature interactions. It should run on Windows and work for sparse data, though
